OptionSelector/OSCategoriesTableViewController.m

```diff
@@ -0,0 +1,125 @@
+//
+//  OSCategoriesTableViewController.m
+//  OptionSelector
+//
+//  Created by Daniel Distant on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OSCategoriesTableViewController.h"
+#import "CQCategory.h"
+#import "OSStuffTableViewController.h"
+
+@interface OSCategoriesTableViewController () <UITableViewDelegate>
+
+@property (nonatomic) NSArray *CQCategories;
+
+
+
+@end
+
+@implementation OSCategoriesTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    CQCategory *dogs = [[CQCategory alloc] init];
+    CQCategory *cats = [[CQCategory alloc] init];
+    CQCategory *birds = [[CQCategory alloc] init];
+    CQCategory *mikes = [[CQCategory alloc] init];
+    
+    self.CQCategories = [[NSArray alloc] init];
```

Comment:
You're setting this property to an empty array, and then to a different array below on line 33, so you can safely remove this line.